Gopkg.toml

```diff
@@ -38,36 +38,36 @@ required = [
 
 [[override]]
   name = "github.com/knative/pkg"
-  # HEAD as of 2019-02-12
-  revision = "1982208dd91a4cc4fea9612e4e72fb109c77ab22"
+  # HEAD as of 2019-03-18
+  revision = "587fb819f44ff27a38a8ceb172058c5214a47911"
 
 [[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.12.5"
+  version = "kubernetes-1.12.6"
```

Comment:
I'd probably mention this in release notes for completeness, but I don't think it's required.